ansible/roles/host-ocp4-installer/tasks/main.yml

```diff
@@ -1,6 +1,11 @@
 - name: Install client and OpenShift Installer binaries
   import_tasks: install_installer.yml
 
+## Open the port for the api
+- name: OpenStack specific requirement - find network port id
+  when: cloud_provider == "osp"
```

Comment:
I think we may want another when here to only do this when ocp_use_single_network is true